packages/components/src/templates/next/components/complex/Infopic/Infopic.stories.tsx

```diff
@@ -75,3 +75,29 @@ export const NoButton: Story = {
     buttonUrl: "",
   },
 }
+
+export const TallImage: Story = {
+  args: {
+    buttonUrl: "",
+    description: "",
+    imageSrc:
+      "https://images.unsplash.com/photo-1724390495674-5f28d72c686f?q=80&w=2500&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
+  },
+}
+
+export const LongImage: Story = {
+  args: {
+    title: "Don't put all your baskets in one egg",
+    description: "",
+    buttonUrl: "",
+    imageSrc: "https://picsum.photos/1200/120",
```

Comment:
Please ensure the images are copyright free yeah

Reply:
I think picsum.photos resizes unsplash pics but let me change to a link from unsplash just to be sure!